Lombiq.Gulp.Extensions.csproj

```diff
@@ -1,33 +1,31 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
     <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**</DefaultItemExcludes>
   </PropertyGroup>
 
   <ItemGroup>
     <None Remove="node_modules\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\Lombiq.Npm.Targets\Lombiq.Npm.Targets.csproj" />
+  </ItemGroup>
+
   <PropertyGroup>
     <InitCommonNodeModulesStampFile>../../node_modules/.install-stamp</InitCommonNodeModulesStampFile>
+    <ESLintConfigFileTargetPath>../../.eslintrc</ESLintConfigFileTargetPath>
```

Comment:
Ennek az itteni eslintrc fájlra kellene hivatkoznia, nem?

Reply:
Nem, a .eslintrc filenak kint kell lennie ott ahova ez hivatkozik, és itt van a logika, hogy kimásolja oda, ha csak a gulp-extension projectben van csak benne

Reply:
Miért kell kint lennie? Ez olyan, mint az editorconfig, hogy ez mappastruktúra alapján töltődik be, és akkor lesz érvényes az egész solutionre, ha a gyökérben van?

Reply:
Amíg nyitott kérdés van, addig ne nyomj a Done-ra légyszi.

Reply:
Igen, pontosan így

Reply:
OK!